api/src/config.rs

```diff
@@ -1260,6 +1267,9 @@ impl TryFrom<&BackendConfig> for BackendConfigV2 {
             "registry" => {
                 config.registry = Some(serde_json::from_value(value.backend_config.clone())?);
             }
+            "noop" => {
```

Comment:
Why do you need `noop` instead of using the `localfs` backend? You can amend `localfs` to work w/o the chunkmap. There is no need to introduce a new backend type for this IMO.